pkg/controller/datavolume-controller.go

```diff
@@ -95,6 +98,7 @@ func NewDataVolumeController(
 		dataVolumesSynced: dataVolumeInformer.Informer().HasSynced,
 		workqueue:         workqueue.NewNamedRateLimitingQueue(workqueue.DefaultControllerRateLimiter(), "DataVolumes"),
 		recorder:          recorder,
+		pvcExpectations:   expectations.NewUIDTrackingControllerExpectations(expectations.NewControllerExpectations()),
 	}
 
 	glog.Info("Setting up event handlers")
```

Comment:
I guess this is ok w/o explicitly passing verbosity. In the importer-controller we would write `glog.V(Vuser).Info(...)`